release-1.8/release_notes_draft.md

```diff
@@ -37,13 +37,6 @@ please check out the [release notes guidance][] issue.
 
 ## **Major Themes**
 
-- The kubernetes workloads API (the DaemonSet, Deployment, ReplicaSet, and
```

Comment:
I think this is definitely a major theme. This should not be removed.

Reply:
I agree with @kow3ns. This is a major theme and something people should know about up front. Can we have SIG Apps updated without removing this theme?

Reply:
Can we just say the following in the main theme?

`The following Kubernetes objects are now part of the new apps/v1beta2 group and version:
  - DaemonSet
  - Deployment
  - ReplicaSet
  - StatefulSet`

Reply:
That's not really a theme. The theme is that we moved these API objects to the apps group so that we can promote them to GA (as a group) in the next release. 
As a user, its something I'd want to know, and if the RBAC section is appropriate here, I think this certainly is.